src/Geno/C++/GUI/Widgets/WorkspaceWidget.cpp

```diff
@@ -128,7 +128,11 @@ void WorkspaceWidget::Show( bool* p_open )
 				{
 					ImGui::CloseCurrentPopup();
 
+					std::filesystem::remove( workspace->location_ );
```

Comment:
We probably want to check if the serialization succeeded before deleting the old file, but since we have no way to tell at the moment, this is OK 😊